### PR TITLE
Added versionadded to etcd sdb module; fixed usage example

### PR DESCRIPTION
**Pull request by whiteinge**

### CODE REVIEW

File: salt/sdb/etcd_db.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 :depends:       python-etcd
 :platform:      all
 
+.. versionadded:: 2015.2
+
 This module allows access to the etcd database using an ``sdb://`` URI. This
```
Comment (rallytime) [inline salt/sdb/etcd_db.py]: I am confused. So was this added in 2015.2 or 2014.1.4 (see line 31 below). This looks like everything is referencing the same file, just in different places with different versions.

**Comment by rallytime** [comment]
ping @whiteinge and @techhat (see comment above)

File: salt/sdb/etcd_db.py (hunk 1 above)
Comment (techhat) [inline salt/sdb/etcd_db.py]: Oops, sorry about that. It was written when 2014.1 was current, but I failed to commit it until today.

Comment (rallytime) [inline salt/sdb/etcd_db.py]: No problem! I'll grab that 2014.1 one and take it out. Thanks!

**Pull request merged**